playkit/src/main/java/com/kaltura/playkit/OnMediaInterceptorListener.java

```diff
@@ -0,0 +1,5 @@
+package com.kaltura.playkit;
+
+public interface OnMediaInterceptorListener {
+    void onApplyMediaCompleted();
```

Comment:
Rename to `onComplete` and add `@Nullable Exception` as parameter.

Reply:
We will change the name to `onComplete`. 
This interface is used by Plugin/Interceptor to notify that it finishes its work. Currently, the `MessageBus` is in charge of error handling for each interceptor. Interceptors not impact the rest of interceptors, and each of interceptors is independent and can't stop the chain of operations. 
So in order to declare exception: do we need to remove `MessageBus` usage from plugin and send any exception from smartlib in `onComplete(Exception)`? If so, do we need to collect all exceptions from a list of Interceptors or just ignore them?

Reply:
The errors reported by plugins in the `MessageBus` are generic and async - they can occur at any time during normal operation of the plugin (not as interceptor).
The errors reported via onComplete(Exception) are specific to this use case: the interceptor/plugin has failed to get the updated/fixed playback URL.

Reply:
ok, i will do it.

Reply:
@litvinenko-kaltura  correction - the implementation should be:
- the listener method name is onComplete()
- it **doesn't** take an error object (for the reasons you mentioned)
- Errors should be logged and potentially posted via the messagebus (like you suggested)